File: SE/ASSIGNEMENT/MODULE 03/STRING/Q2.cpp
// Q(2)Write a program in C to separate individual characters from a string.....

#include<stdio.h>

main()
{
	char a[50];
	int i;
	
	printf("enter the character:");				//input.. a
	gets(a);
	
	printf("------------------------------\n");
	
	printf("print the character in separate..");
	
	for(i=0;a[i]!='\0';i++)
	{
		if(a[i]==' ')
		
			continue;
		
		else
			printf("\n %c",a[i]);				//print the separate input
		
	}
	
}

File: PYTHON PROGRAMMING/ASSIGNMENTS/Module – 2 (Fundamentals of python)/Q2.py
#Write a Python program to get the Factorial number of given number.

num=int(input("Enter the number: "))

            # give fact==1
fact=1

            # used for loop from 1 to user input..
for i in range(1,num+1):

            # fact*=i...
    fact=fact*i

print("factorial: ",fact)